<launch>
    <node name="tp_controller" pkg="iauv_kinematic_control" type="task_priority_controller" output="screen">
        <rosparam command="load" file="$(find iauv_manipulation)/config/task_priority.yaml"/>
        <remap from="robot_description" to="/girona1000/robot_description"/>
        <remap from="tp_controller/joint_states" to="/girona1000/joint_states"/>
        <remap from="tp_controller/odometry" to="/girona1000/navigator/odometry"/>
        <remap from="tp_controller/base/command" to="/girona1000/controller/body_velocity_req"/>
        <remap from="tp_controller/bravo/command" to="/girona1000/bravo/joint_velocity_controller/command"/>
    </node>
</launch>